docs/best-practices.md

```diff
@@ -0,0 +1,380 @@
+# Best Practices and Tips
+
+This page is about to give contributors a quick overview of tips and best practices for 
+writing in asciidoc. It is for sure not complete nor covers all possibilities, but it's
+intention is to quickly speed up. For a complete reference see the
+[Asciidoctor Documentation](https://asciidoctor.org/docs/).
+
+:toc:
+
+## Initial Reading
+
+Here is a good starting point to get a quick Antora overview [`Three Core Antora Concepts`](https://matthewsetter.com/antoras-three-core-concepts/)
+
+## File Locations
+
+* All documents are written into directory `modules`/`module_name`/`pages`/`<path>`
+* All images are written into directory `modules`/`module_name`/`assets/images`/`<path>`
+* All examples are written into directory `modules`/`module_name`/`examples`/`<path>`
+
+When using paths to include, you might need to use the `module_name` when linking to another module,
+but you must not use `pages`, `assets/images` or `examples` as path component, only `<path>`.
+See the examples in the relevant sections.
+
+## Links
+
+In a nutshell, there are two kind of links.
+- External links
+- Internal links (referencing content inside the documentation)
+
+
+### External Links
+
+Reference: [`Links`](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links)
+
+A `link` is written in following example style: `http(s)://domain/#section[Printed Name]`
```

Comment:
How about, instead of "A `link` is written in following example style", use: "A `link` follows this style"